### PR TITLE
Integrated text voicer into voice recognition showcase prototype.

### PR DESCRIPTION
**Pull request by marjasi**

### CODE REVIEW

**Comment by EdvardasDlugauskas** [comment]
Edit: oh, it only adds the project to solution, I see

How exactly does it work now? "What is this" captures the current image (like capture button), like we discussed?

File: ShopLens/VoiceRecognition/VoiceRecognitionTest.cs [hunk 1]
```diff
@@ -65,37 +71,45 @@ public void SpeechRecognizer_SpeechRecognized(object sender, SpeechRecognizedEve
 
             if (speechResult == helloCmd)
```
Comment (EdvardasDlugauskas) [inline ShopLens/VoiceRecognition/VoiceRecognitionTest.cs:72]: Possible way to make this code shorter - use 'switch' statement and assign the response to a variable, and just use that assigned variable at the end (only one place where you call `.Text +=` and `.SayMessage` this way. Something like this:

```csharp
string response = "";
switch (speechResult):
{
    case(helloCmd):
        response = helloRsp;
        break;
     ....
}

CommandOutputBox.Text += response;
textVoicer.SayMesage(response);

**Review by EdvardasDlugauskas — APPROVED** [review]
Look like great code to me 👍

File: ShopLens/VoiceRecognition/VoiceRecognitionTest.cs (hunk 1 above)
Comment (marjasi) [reply ShopLens/VoiceRecognition/VoiceRecognitionTest.cs:72]: That's a good point, I'll change it.

**Pull request merged**